test/shared_test/lib_battery_lifetime_nmc_test.cpp

```diff
@@ -693,3 +698,168 @@ TEST_F(lib_battery_lifetime_nmc_test, replaceBatteryTest) {
     EXPECT_NEAR(s.average_range, 0, tol);
     EXPECT_NEAR(s.n_cycles, 0, tol);
 }
+
+TEST_F(lib_battery_lifetime_nmc_test, BTMSTest) {
+    std::ifstream in("/Users/dguittet/Desktop/batt_soc_NMCGraphite_1_0.5hr_20yrs.txt");
```

Comment:
Need to remove local paths from the tests.

Reply:
Oops this was erroneously committed. Let me remove this